app/migrations/Version20170829081048.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\AbstractMauticMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20170829081048 extends AbstractMauticMigration
+{
+    
+    /**
+     * @param Schema $schema
+     *
+     * @throws SkipMigrationException
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function preUp(Schema $schema)
+    {
+        $table = $schema->getTable($this->prefix.'email_stats');
+
+        if ($table->hasColumn('is_replyed')) {
+            throw new SkipMigrationException('Schema includes this migration');
+        }
+    }
+    
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE '.$this->prefix.'email_stats ADD is_replyed TINYINT(1) NOT NULL');
```

Comment:
@alanhartless won't like to modify the `email_stats` table since it can be massive for some Mautic installations and can break the upgrade process.

My suggestion is to create new `email_replies` table where we could store more information about the replies. Like what contact replied and what message did he sent so we could show it in his/hers timeline. And we could add additional features based on it. For example campaign condition which could check whether the reply contained some word. What do you think?